cdc/owner.go

```diff
@@ -1021,6 +1002,20 @@ func (o *ownerImpl) Run(ctx context.Context, tickTime time.Duration) error {
 			if !o.IsOwner(ctx) {
 				continue
 			}
+
+			// Do something initialize when the capture becomes an owner.
+			once.Do(func() {
+				// Start a routine to keep watching on the liveness of
+				// processors.
+				o.startProcessorInfoWatcher(ctx)
+
+				// When an owner crashed, its processors crashed too,
+				// clean up the tasks for these processors.
+				if err := o.cleanUpStaleTasks(ctx); err != nil {
```

Comment:
Should `cleanUpStaleTasks` be called every time when a capture becomes owner?
Consider the following scenario:
1. start two captures
2. resign owner twice
3. kill -9 `the pid of current owner`

Found some tables are not re-balanced

Reply:
Good catch, I will fix it.